apps/studio/prisma/schema.prisma

```diff
@@ -26,68 +26,77 @@ model VerificationToken {
   expires    DateTime
 }
 
-model Post {
-  id           String       @id @default(cuid())
-  title        String?      @db.Text
-  content      String       @db.Text
-  contentHtml  String       @db.Text
-  authorId     String
-  author       User         @relation(fields: [authorId], references: [id], onDelete: Cascade)
-  images       String[]
-  likes        LikedPosts[]
-  // This features the self-relation Prisma concept, specifically a one-to-many self relation.
-  // https://www.prisma.io/docs/concepts/components/prisma-schema/relations/self-relations
-  parentPostId String?
-  // The parent post of this post, if it is a reply.
-  // @relation("PostToParent") is the name of the relation. See relation documentation below for more info.
-  parent       Post?        @relation("PostReplies", fields: [parentPostId], references: [id])
-  // The replies to this post.
-  replies      Post[]       @relation("PostReplies")
-
-  // To return `Date`s intact through the API we use transformers
-  // https://trpc.io/docs/data-transformers
-  // This is unique so it can be used for cursor-based pagination
-  createdAt DateTime  @default(now())
-  updatedAt DateTime  @default(now()) @updatedAt
-  deletedAt DateTime? @db.Timestamp
-
-  @@index([parentPostId])
-  @@index([parentPostId, deletedAt, id])
-  @@index([deletedAt])
-  @@index([createdAt])
+model Resource {
+  id             String       @id @default(cuid())
+  name           String
+  siteId         String
+  parentId       String?
+  lastEditTime   DateTime?
+  lastEditUserId String
+  blobId         String?      @unique
+  variant        ResourceType
```

Comment:
suggest removing `variant` here and let the code decide by presence of `blobId`. reasons why this makes sense: 1 less thing to set in our db + product still young, assumptions can change later on.

wdyt? (this isn't a big deal though so happy to keep as well)

Reply:
Now that you mentioned it, seems like it makes sense to remove variant for now, since there is a direct correlation between blobId being null and variant being folder.
Should also be possible to add such a field in a future migration if needed.